src/features/verify/verifySlice.ts

```diff
@@ -0,0 +1,45 @@
+import { createSlice, type PayloadAction} from '@reduxjs/toolkit';
+
+export type TokenPayload = {
+    uid: string | [string],
+    token: string | [string],
+}
+
+type VerifyState = {
+    token: TokenPayload | null,
+    isLoading: 'idle' | 'pending',
+    error: TokenPayload | null,
+}
+
+const initialState: VerifyState = {
+    token: null,
+    isLoading: 'idle',
+    error: null,
+}
+
+export const verifySlice = createSlice({
+    name: 'verify',
+    initialState,
+    reducers: {
+        verifing: ( state, action: PayloadAction<TokenPayload>) => {
```

Comment:
Всегда лучше называть через глагол в инфинитиве + существительное
```suggestion
        verifyUser: ( state, action: PayloadAction<TokenPayload>) => {
```